web/src/components/OpeningHours.tsx

```diff
@@ -24,8 +24,9 @@ const Content = styled.div`
 const TitleContainer = styled.div`
   display: flex;
   flex: 1;
-  font-weight: 700;
   justify-content: space-between;
+  flex-wrap: wrap;
```

Comment:
The space between Wegbeschreibung and the address is different from the space between Öffnungszeiten and Vorübergehend geschlossen when its wrapped.
I think you can use `gap: 12px;` to solve it 🙂.
<img src="https://github.com/user-attachments/assets/f8175e53-dc1e-49bf-8cb5-7863d8132083" width="auto" />